doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,229 @@
+>>> from sympy.core import Rel
+>>> from sympy import symbols, Integer, pi, solve
+>>> x = symbols('x')
+>>> inequality1 = Rel(x, Integer(1)/Integer(3), 'gt')
+>>> inequality1
+x > 1/3
+>>> inequality2 = Rel(x**2, pi, '<=')
+>>> inequality2
+x**2 <= pi
+>>> solve([inequality1, inequality2], x)
+(1/3 < x) & (x <= sqrt(pi))
+```
+
+### Parse a string representing each inequality
+
+Parse a string representing each inequality into a form that SymPy can
+understand, then apply solve() to the list of parsed inequalities. This approach
+is convenient if you are programmatically reading in a string for each
+inequality. If you are creating the expression yourself, we [recommend against
+parsing a
+string](https://github.com/sympy/sympy/wiki/Idioms-and-Antipatterns#strings-as-input).
+
+```py
+>>> from sympy import parse_expr, pi, solve
+>>> from sympy.abc import x
+>>> inequality1 = 'x >= 0'
+>>> inequality2 = 'x**2 <= pi'
+>>> inequalities = [inequality1, inequality2]
+>>> inequalities_parsed = [parse_expr(inequality) for inequality in inequalities]
+>>> inequalities_parsed
+[x >= 0, x**2 <= pi]
+>>> solve(inequalities_parsed, x)
+(0 <= x) & (x <= sqrt(pi))
+```
+
+## Use the solution result
+
+A common way to use the solution result is to extract the bounds for the symbol
+(variable). For example, for a solution of $0 < x < \sqrt{pi}$, you might want
+to extract $0$ and $\sqrt{pi}$.
+
+### Extract relational atoms
+
+You can decompose a set of relations which is joined by `^` (or) or `&` (and)
+into individual relations using relational atoms. Using {any}`canonical
+<sympy.core.relational.Relational.canonical>` will put order each relation so
+the symbol is on the left, so you can take the right-hand side {any}`rhs
+<sympy.core.relational.Relational.lhs>` to extract the constants:
+
+```py
+>>> from sympy import symbols, solve, Integer, pi
+>>> from sympy.core.relational import Relational
+>>> x = symbols('x')
+>>> one_third = Integer(1)/Integer(3)
+>>> eq = solve([x >= one_third, x**2 <= pi], x)
+>>> relations = [(i.lhs, i.rel_op, i.rhs) for i in [i.canonical for i in eq.atoms(Relational)]]
+>>> # Sorting relations just to ensure consistent list order for docstring testing
+>>> relations_sorted = sorted(relations, key=lambda x: float(x[2]))
+>>> print(relations_sorted)
+[(x, '>=', 1/3), (x, '<=', sqrt(pi))]
+```
+
+### Extract arguments
+
+The {any}`args <sympy.core.basic.Basic.args>` of a solution set are the
+individual relations, so you can extract the constants from the left- or
+right-hand side of the `args`:
+
+```py
+>>> from sympy import symbols, solve, Integer, pi
+>>> x = symbols('x')
+>>> eq = solve([x >= Integer(1)/Integer(3), x**2 <= pi], x)
+>>> eq.args
+(1/3 <= x, x <= sqrt(pi))
+>>> constants = []
+>>> for arg in eq.args:
+...     if arg.lhs == x:
+...         constants.append(arg.rhs)
+...     else:
+...         constants.append(arg.lhs)
+>>> constants
+[1/3, sqrt(pi)]
+```
+
+## *Tradeoffs (speed vs. accuracy, etc.) for function*
```

Comment:
Are `check=False` and `simplify=False` worth mentioning here as ways to potentially speed up `solve` for a system of inequalities?

I am also wondering whether it makes sense to have a page that describes such options for `solve` in general, rather than discussing those options on multiple solving pages within doc/guides/solving.

Reply:
I don't know if check and simplify have any effect for inequalities.

Reply:
Thanks.